.env.example

```
@@ -34,7 +37,7 @@ SPRING_PROFILES_ACTIVE=dev
 SPRING_DATASOURCE_URL=jdbc:postgresql://<neon-host>:5432/<dbname>?sslmode=require
 SPRING_DATASOURCE_USERNAME=<neon-username>
 SPRING_DATASOURCE_PASSWORD=<neon-password>
-JWT_SECRET=<your-jwt-secret>
+JWT_SECRET=<your-jwt-secret-at-least-32-bytes-long>
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The JWT_SECRET key is duplicated

(DuplicatedKey)